cypress/e2e/tests/rbac/architect.test.ts

```diff
@@ -35,9 +42,20 @@ describe(["@tier2", "@rhsso"], "Architect RBAC operations", function () {
 
     before("Creating RBAC users, adding roles for them", function () {
         login();
+        AssessmentQuestionnaire.enable(fileName);
+        // Navigate to stakeholders control tab and create new stakeholder
+        const stakeholder = new Stakeholders(data.getEmail(), data.getFullName());
+        stakeholder.create();
+        cy.wait(2 * SEC);
+
+        stakeholdersList.push(stakeholder);
+        stakeholdersNameList.push(stakeholder.name);
+
```

Comment:
You're creating a stakeholder and adding it and its name to an array but the test doesn't use any of that.

Reply:
I need it to run assessment, otherwise I can't run it (list of stakeholders is a mandatory parameter) and then some expected buttons in UI are not available.

Reply:
Same thing as I don't get why stakeholders list is mandatory and why creator of this code used only NAMES of stakeholders so I need to create 2 arrays, one with stakeholders and second with their names, why it doesn't take list of stakeholders themselves... but that was too much to refactor in THIS PR.